engine/common/imagelib/img_bmp.c

```diff
@@ -174,7 +180,7 @@ qboolean Image_LoadBMP( const char *name, const byte *buffer, fs_offset_t filesi
 		break;
 	}
 
-	estimatedSize = ( buf_p - buffer ) + ( image.width + padSize ) * image.height * ( bhdr.bitsPerPixel >> 3 );
+	estimatedSize = ( buf_p - buffer ) + image.width * image.height * ( bhdr.bitsPerPixel >> 3 );
```

Comment:
Why did you remove padding here?

Reply:
It causes invalid estimated size calculation on images which dimensions is not power of 2. And therefore throws error.